eox_nelp/mfe_config_api/v1/views.py

```diff
@@ -0,0 +1,44 @@
+"""The generic views for MFE_CONFIG API. Nelp flavour."""
+
+import json
+
+from django.conf import settings
+from django.http import JsonResponse
+from eox_theming.configuration import ThemingConfiguration as theming
+from rest_framework import status
+
+from eox_nelp.edxapp_wrapper.mfe_config_view import MFEConfigView
+
+
+class NelpMFEConfigView(MFEConfigView):
+    """
```

Comment:
This doc string could be improved, for example, https://github.com/eduNEXT/eox-tenant/blob/master/eox_tenant/api/v1/viewsets.py#L59

Reply:
[9b2ec3d](https://github.com/eduNEXT/eox-nelp/pull/23/commits/9b2ec3df2ce6ff82d5b6513e288af24df05c2698)